addon/file.js

```diff
@@ -1,14 +1,12 @@
-import { registerWaiter } from '@ember/test';
-import { DEBUG } from '@glimmer/env';
-
 import { assert } from '@ember/debug';
-import EmberObject, { set, computed } from '@ember/object';
 import FileReader from './system/file-reader';
 import HTTPRequest from './system/http-request';
 import RSVP from 'rsvp';
 import uuid from './system/uuid';
+import { tracked } from '@glimmer/tracking';
+import { buildWaiter } from 'ember-test-waiters';
```

Comment:
Should this be `import { buildWaiter } from '@ember/test-waiters';` now?

Reply:
Good catch – thanks! Fixed in 1ee1569c34f3c51f1fbe0a4d5966e318a20c5aab